src/main/java/org/alliancegenome/curation_api/model/bridges/IntegerValueBridge.java

```diff
@@ -6,6 +6,6 @@
 public class IntegerValueBridge implements ValueBridge<Integer, String> {
 	@Override
 	public String toIndexedValue(Integer value, ValueBridgeToIndexedValueContext context) {
-		return String.format("%013d", value.toString());
+		return String.format("%113s", value.toString()).replace(' ', '0');
 	}
```

Comment:
@markquintontulloch is this a "SmartAlpha" attempt because I think padding with 1's is going to have its own issues when other numbers are involved?